src/Bicep.Core.IntegrationTests/TestFrameworkTests.cs

```diff
@@ -36,19 +36,22 @@ public void TestFramework_is_disabled_unless_feature_is_enabled()
 ");
             result.Should().HaveDiagnostics(new[] {
                 ("BCP348", DiagnosticLevel.Error, "Using a test declaration statement requires enabling EXPERIMENTAL feature \"TestFramework\"."),
+                ("BCP091", DiagnosticLevel.Error, @"An error occurred reading file. Could not find file 'C:\path\to\test1.bicep'.")
```

Comment:
There are some extra test failures on the linux and OS X test suites because the path doesn't match:

```
Expected AssertionExtensions to satisfy all inspectors, but some inspectors are not satisfied:
	At index 1:
		Expected message to be "An error occurred reading file. Could not find file 'C:\path\to\test1.bicep'." but it was "An error occurred reading file. Could not find file '/path/to/test1.bicep'."

With diagnostics:
1| 
2| test test1 'test1.bicep' = {}
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ [BCP348 (Error)] Using a test declaration statement requires enabling EXPERIMENTAL feature "TestFramework".
              ~~~~~~~~~~~~~ [BCP091 (Error)] An error occurred reading file. Could not find file '/path/to/test1.bicep'.
3| 
   at FluentAssertions.Execution.LateBoundTestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.CollectingAssertionStrategy.ThrowIfAny(IDictionary`2 context)
   at Bicep.Core.UnitTests.Assertions.DiagnosticAssertions.DoWithDiagnosticAnnotations(BicepSourceFile bicepFile, IEnumerable`1 diagnostics, Action`1 action) in /home/runner/work/bicep/bicep/src/Bicep.Core.UnitTests/Assertions/DiagnosticAssertions.cs:line 60
   at Bicep.Core.UnitTests.Assertions.CompilationResultAssertions.DoWithDiagnosticAnnotations(Action`1 action) in /home/runner/work/bicep/bicep/src/Bicep.Core.UnitTests/Assertions/CompilationResultAssertions.cs:line 60
   at Bicep.Core.UnitTests.Assertions.CompilationResultAssertions.HaveDiagnostics(IEnumerable`1 expectedDiagnostics, String because, Object[] becauseArgs) in /home/runner/work/bicep/bicep/src/Bicep.Core.UnitTests/Assertions/CompilationResultAssertions.cs:line 76
   at Bicep.Core.IntegrationTests.TestFrameworkTests.TestFramework_is_disabled_unless_feature_is_enabled() in /home/runner/work/bicep/bicep/src/Bicep.Core.IntegrationTests/TestFrameworkTests.cs:line 37
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
```

There's no version of this error that will match on both Windows and Linux, so I would recommend passing multiple files to `CompilationHelper.Compile()` so that 'test1.bicep' does exist. (A Bicep template file can be empty, so you can use an empty string for that.) This test is really only concerned with validating that the feature flag is checked.